app/Livewire/Project/Shared/EnvironmentVariable/All.php

```
@@ -65,8 +78,25 @@ public function instantSave()
+    }
+
+    private function getEnvironmentVariables(bool $isPreview, bool $withSearch = true): Collection
+    {
+        $query = $isPreview
+            ? $this->resource->environment_variables_preview()
+            : $this->resource->environment_variables();
+
+        $query->orderByRaw("CASE WHEN is_required = true AND (value IS NULL OR value = '') THEN 0 ELSE 1 END");
+
+        if ($withSearch && $this->searchTerm() !== '') {
+            $query->whereRaw('LOWER(key) LIKE ?', ['%'.Str::lower($this->searchTerm()).'%']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method 'getEnvironmentVariables'. (undefined)

(StaticAccess)